### DISCUSSION TITLE
What do you think about the `planning/` repo?

### DISCUSSION THREAD

**Discussion by choldgraf** [2i2c-org/team-compass#89] [General]
I just added a `planning/` repository, with the goal of making it easier to track our deliverables that are actually in *other* GitHub organizations (e.g., for JupyterHub specifically):

https://github.com/2i2c-org/planning

The README has some explanation of how the repository could be used.
I'll try it out myself over the coming weeks and see if this is a helpful way for us to standardize some of the information in our project boards.
Would also love feedback from the @2i2c-org/tech-team if they find it useful.

**Comment by damianavila** [comment]
I am not sure if `planning` is the best name for that repo (somehow implies a general planning place and not a planning place for deliverables living outside 2i2c org) but let's try it out for some time and we can eventually change the name (or not) :wink:
